canonical-web-and-design/charmhub.io
# Remove featured and other filters

## Done

- Render entity cards on the front-end
- Handle platform filter on the front-end

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Change platforms and check that you see what is expected and that the URL is updated


## Issue / Card

Fixes #573
